lib/ansible/executor/playbook_executor.py

```diff
@@ -247,6 +247,9 @@ def _generate_retry_inventory(self, retry_path, replay_hosts):
         re-running on ONLY the failed hosts.  This may duplicate some variable
         information in group_vars/host_vars but that is ok, and expected.
         '''
+        dirname = os.path.dirname(retry_path)
+        if not os.path.exists(dirname):
+            os.makedirs(dirname)
```

Comment:
@bcoca cool, thanks! I had a look around the code before making the change and saw a few `makedirs` around, but I was sure there had to be some util function like that :)  I'll update the branch this evening!